frontend/model/contracts/group.js

```diff
@@ -1359,7 +1359,7 @@ sbp('chelonia/defineContract', {
         archSentOrReceivedPayments.sent = [...sortPayments(newSentOrReceivedPayments.sent), ...archSentOrReceivedPayments.sent]
         archSentOrReceivedPayments.received = [...sortPayments(newSentOrReceivedPayments.received), ...archSentOrReceivedPayments.received]
 
-        const archPaymentsKey = `payments/${period}/${username}/${contractID}`
+        const archPaymentsKey = `payments/${username}/${period}/${contractID}`
```

Comment:
So the username was being saved to the key...?

How does this materially change anything?